tools/update-git-upstream

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu
+
+upstream_url="https://github.com/zulip/zulip.git"
+if ! git remote | grep -q upstream; then
```

Comment:
Should be `grep -qx`, or in pure Bash,

```bash
remotes="$(git remote)"
if [[ $'\n'"$remotes"$'\n' != *$'\nupstream\n'* ]]; then
```

Reply:
I'm not sure the pure-bash version is very readable, so I'll update to `grep -qx`.